test/parallel/test-http-set-timeout-server.js

```diff
@@ -117,20 +117,25 @@ test(function serverRequestNotTimeoutAfterEnd(cb) {
 
 test(function serverResponseTimeoutWithPipeline(cb) {
   let caughtTimeout = '';
+  let secReceived = false;
   process.on('exit', function() {
     assert.strictEqual(caughtTimeout, '/2');
   });
   const server = http.createServer(function(req, res) {
```

Comment:
I don't know if I can wrap the `http.createServer()` callback either as I'm not sure it's going to be called always 3 times.